docs/distributed-tracing-guide.asciidoc

```diff
@@ -62,3 +62,22 @@ please see the MDN page on https://developer.mozilla.org/en-US/docs/Web/HTTP/COR
 
 NOTE: To make sure all components in a distributed trace are included,
 the sampling rate of backend agents might be affected by the sampling rate of the RUM agent.
+
+[float]
+[[dynamic-html-doc]]
+=== Dynamically-generated HTML
+To inculde the transactions generated by the back-end agents for the dynamically-generated initial HTML document, 
+you should provide the following configuration options while initializing the RUM agent:
+
+[source,js]
+----
+var apm = initApm({
+    ...
+    distributedTracingPageLoadTraceId: <trace-id>,
+    distributedTracingPageLoadSpanId: <span-id>,
+    distributedTracingPageLoadSampled: <is-sampled>
+})
+----
+
+Our back-end agents provide APIs to extract these information, 
+please refer to https://www.elastic.co/guide/en/apm/agent/index.html[the documentation of the agent] for more detail.
```

Comment:
Update link to use cross document links instead of being hardcoded

```suggestion
please refer to {apm-agents-ref}/index.html[the documentation of the agent] for more detail.
```